app/src/main/java/org/kinecosystem/kinit/repository/TasksRepo.kt

```diff
@@ -0,0 +1,97 @@
+package org.kinecosystem.kinit.repository
+
+import android.content.Context
+import android.databinding.ObservableBoolean
+import android.text.TextUtils
+import com.google.gson.Gson
+import org.kinecosystem.kinit.KinitApplication
+import org.kinecosystem.kinit.model.TaskState
+import org.kinecosystem.kinit.model.earn.*
+import org.kinecosystem.kinit.util.ImageUtils
+import javax.inject.Inject
+
+private const val QUESTIONNAIRE_ANSWERS_STORAGE = "kin.app.task.chosen.answers"
+private const val TASK_STORAGE = "kin.app.task"
+private const val TASK_KEY = "task"
+private const val TASK_STATE_KEY = "task_state"
+private const val SHOW_CAPTCHA_KEY = "show_captcha"
+
+class TasksRepo(categoryId:String, var task: Task?) {
```

Comment:
This holds a single task - so would rename it to TaskRepo or TaskWrapper or maybe we can think of a different better name for it.